include/screen.h

```diff
@@ -0,0 +1,28 @@
+#ifndef SCREEN_H
+#define SCREEN_H
+
+#include <iostream>
+#include <SDL2/SDL.h>
+#include <GL/glew.h>
+
+class Screen {
+private:
+  int m_height;
+  int m_width;
+  std::string m_title;
+  SDL_Window* m_window;
+  SDL_GLContext m_glContext;
+  bool m_windowOpen;
+
+  Screen() {} // Default private constructor
+
+public:
+  Screen(int height, int width, const std::string& title);
+  virtual ~Screen();
```

Comment:
Technically it will make the compiler generate a vtable for your class, making the memory footprint slightly larger, but, practically, no. I'd still say remove it until you do make it the base of something.